tests/WE2E/run_we2e_tests.py

```diff
@@ -40,14 +40,21 @@ def run_we2e_tests(homedir, args) -> None:
 
     # Set up logging to write to screen and logfile
     setup_logging(debug=args.debug)
-
+    logging.debug(f"Arguments to run_we2e_tests():\n{args}")
     # Set some important directories
     ushdir = Path(homedir, "ush")
 
     # Set some variables based on input arguments
     run_envir = args.run_envir
     machine = args.machine.lower()
 
+    # Derecho requires long delay between calls to rocotorun due to system-level cacheing of
+    # job statuses
+    if machine=="DERECHO":
```

Comment:
On line 49, `machine` is set to `args.machine.lower()`.  Since `machine` is converted to lower case, should `DERECHO` be `derecho`?

```suggestion
    if machine=="derecho":
```